src/Shared/NativeMethodsShared.cs

```diff
@@ -503,6 +503,16 @@ internal static bool IsBSD
 #endif
         }
 
+        private static readonly bool s_is64BitProcess = Marshal.SizeOf(NullIntPtr) == 8;
```

Comment:
IntPtr.Size == 8?

Reply:
This is a copy of another file that checks the 64bit process.
If you have a better practice will replace it.

Reply:
Tensorflow flow by default builds with 32 bit version of msbuild with the 32 bit c++ toolchain and fails.
Moving to 64 bit toolchain fixes the issue, but we get this, which is a problem.
Here is the link.
https://support.microsoft.com/en-us/help/942589/a-32-bit-application-cannot-access-the-system32-folder-on-a-computer-t